sdk/src/error.rs

```diff
@@ -272,6 +278,10 @@ pub enum Error {
 
     #[error("prerelease content detected")]
     PrereleaseError,
+
+    #[error(transparent)]
+    #[cfg(feature = "pdf")]
+    PdfError(#[from] pdf_utils::Error),
```

Comment:
hmmm, maybe we should have a class of AssetIO errors with subclasses for each type. I'm kind of nervous about adding top level errors for every type of supported asset structurally.

Reply:
The current model is to use Error::InvalidAsset() and pass a string giving the particular problem

Reply:
Ah, that makes sense. I'll switch to using `InvalidAsset`. Thanks!